### PR TITLE
Fast api setup

### PR DESCRIPTION
**Pull request by kerembaskaya**

### CODE REVIEW

File: summarizer/main.py [hunk 1]
```diff
@@ -0,0 +1,87 @@
...
+        "Google Reviews Summarizer is working! See the doc at" "/api/v1/docs"
+    )
+    return {"success": True, "message": message}
+
+
+# The endpoint for Google API Crawler
+# TODO: It's not important but limit parameter should put in the backhand.
+@v1.get("/google-crawl-reviews", tags=["Google-Crawl"])
+async def fetch_reviews_with_google_api(
+    query: str, limit: int = Query(le=5, ge=1)
```
Comment (osmanbaskaya) [inline summarizer/main.py]: Maybe default for the limit? 5

File: summarizer_resources/utils/customized_logging.py [hunk 2]
```diff
@@ -0,0 +1,60 @@
...
+        "handlers": {
+            "default": {
+                "level": "INFO",
+                "formatter": "standard",
+                "class": "logging.StreamHandler",
+                "stream": "ext://sys.stdout",  # Default is stderr
+            },
+        },
+        "loggers": {
+            "common": {
```
Comment (osmanbaskaya) [inline summarizer_resources/utils/customized_logging.py]: Unnecessary stuff should be removed and `summarizer` should be added

**Review by osmanbaskaya — COMMENTED** [review]
A few suggestions

File: summarizer/main.py (hunk 1 above)
Comment (kerembaskaya) [reply summarizer/main.py]: Since Google has already made that limitation, I didn't.  Should I add a limit according to the situation?

**Pull request merged**